README.md

```diff
@@ -1,2 +1,52 @@
-# wallet-connect-sdk
-A Wallet Connect SDK for ecosystem tools
+# Wallet Connect SDK
+A Coz's [Wallet Connect](https://walletconnect.org/) 2.0 SDK for ecosystem tools.
+
+**Wallet Connect integration is a secure solution when creating a Decentralized Application that needs user's Account
+authentication. This SDK comes in handy to help on such integration.** 
+
+- [Why do I need Wallet Connect in my dApp?](#why-do-i-need-wallet-connect-in-my-dapp)
+- [Usage example video](#usage-example-video)
+- [How does it work?](#how-does-it-work)
+- [Installation and Setup Guide](#installation-and-setup-guide)
+- [Using with React.js](packages/wallet-connect-sdk-react/README.md)
+- [Using without React.js](packages/wallet-connect-sdk-core/README.md)
+- [How to test my dApp?](#how-to-test-my-dapp)
+
+## Why do I need Wallet Connect in my dApp?
+Almost every dApp needs user's authentication to send a signed transaction to the blockchain, the most common scenery is
+when the client-side application needs to call a SmartContract method that needs the user's Account.
+
```

Comment:
Almost every dApp needs user's authentication to send a signed transaction to the blockchain. From minting tokens to making a simple transfer, user's must always sign their transactions whenever the client-side application needs to call a SmartContract method that requires the user's Account.

Reply:
Fixed